cmd/adapter/main.go

```diff
@@ -111,10 +113,14 @@ func (a *Adapter) makeProvider(ctx context.Context, globalHTTPTimeout time.Durat
 	mgr, err := ctrl.NewManager(cfg, ctrl.Options{
 		MetricsBindAddress: metricsBindAddress,
 		Scheme:             scheme,
-		Namespace:          namespace,
-		LeaseDuration:      leaseDuration,
-		RenewDeadline:      renewDeadline,
-		RetryPeriod:        retryPeriod,
+		NewCache: func(config *rest.Config, opts ctrlcache.Options) (ctrlcache.Cache, error) {
```

Comment:
Can't you just use `Cache` in here instead of `NewCache`?

Reply:
Actually I think we should use `Cache` in here for sure: https://github.com/kubernetes-sigs/controller-runtime/blob/30eae58f1b984c1b8139dd9b9f68dd2d530ed429/pkg/manager/manager.go#L114-L126

Reply:
Resolved, thanks!